openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip_jsoniter/src/main/scala/Main.scala

```diff
@@ -0,0 +1,31 @@
+object Main extends App {
+  /*
```

Comment:
**question:** Is this commented code useful for anything?

Reply:
No, these are just c+p artifacts that I should've caught 🤦

Reply:
Removed the comments